web/app/scripts/state/mapstate-service.js

```diff
@@ -45,11 +45,11 @@
         }
 
         /**
-         * Get zoomlevel
+         * Get zoomlevel - default to 5
          */
         function getZoom() {
-            $log.debug('Getting stored map zoom...');
-            return zoom;
+            $log.debug('Getting stored map zoom: ' + JSON.stringify(zoom));
+            return zoom || 5;
```

Comment:
I'm not sure how travis was passing before, but without this, map state wasn't being reset properly on return to the map page